src/aosm/azext_aosm/generate_nsd/nsd_generator.py

```diff
@@ -142,183 +102,125 @@ def config_group_schema_dict(self) -> Dict[str, Any]:
             "If you wish to use a system assigned identity, set this to a blank string."
         )
 
-        if self.config.multiple_instances:
-            deploy_parameters = {
-                "type": "array",
-                "items": {
-                    "type": "object",
-                    "properties": self.deploy_parameters["properties"],
-                },
-            }
-        else:
-            deploy_parameters = {
-                "type": "object",
-                "properties": self.deploy_parameters["properties"],
+        properties = {
```

Comment:
Be nice to have an example comment of what sort of thing this creates, I've just done rather a lot of jumping around code to work out what's happening generally in the nsd_generator and nf_ret classes

Reply:
I've added output to the test directory and pointed at that.